app/controllers/api/v2/hosts_controller.rb

```diff
@@ -123,7 +123,7 @@ def destroy
       eos
 
       def status
-        render :json => { :status => @host.host_status }.to_json if @host
+        render :json => { :status => @host.get_status('HostStatus::ConfigurationStatus').to_label }.to_json if @host
```

Comment:
I wanted to avoid bigger API changes, I suppose status should be global status and we should introduce new actions like configuration status but that would break V2 compatibility. So is this a good time for V3 and should we mix it into this PR?